packages/core/admin/admin/src/hooks/useRolesList/index.js

```diff
@@ -17,7 +17,7 @@ const useRolesList = (shouldFetchData = true) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [shouldFetchData]);
 
-  const fetchRolesList = async () => {
+  const fetchRolesList = useCallback(async () => {
```

Comment:
`fetchRolesList` was used with a ref to memoize, so I thought I might as well memoize it. I can put it back if needed

Reply:
I personally don't really like to memoize without a good reason. It obfuscates code and sometimes it's not necessarily a perf improvement 😊 . When necessary and justified I would totally agree to keep them. But since we're not able to see a real impact, except complicating the codebase, I'm not sure about the need to use it :D